<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Typedeconge">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QGroupBox {
	border:2px solid #1921ab;
	border-radius:7px;
	display:inline-block;
	cursor:pointer;
	font-family:Arial;
	font-size:11px;
	border-size:4px;
	font-weight:bold;
	font-style:italic;
	padding:0px 5px;
	text-decoration:none;
}</string>
   </property>
   <property name="title">
    <string>Type de Congé : </string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>19</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="ICSS">
       <property name="text">
        <string>CSS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ICE">
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ICP">
       <property name="text">
        <string>CP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ICR">
       <property name="text">
        <string>CR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="Titre_Conge">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Demande De Congé</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>06h -&gt; 14h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14h -&gt; 22h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22h -&gt; 06h</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>308</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Icons/Icons/calandrier1-removebg-preview.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>N° Matricule       : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Date début ...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>451</width>
     <height>80</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">.QGroupBox {
	border:2px solid #1921ab;
	border-radius:7px;
	display:inline-block;
	cursor:pointer;
	font-family:Arial;
	font-size:11px;
	border-size:4px;
	font-weight:bold;
	font-style:italic;
	padding:0px 5px;
	text-decoration:none;
	text-shadow:2px 1px 0px #2f6627;
}</string>
   </property>
   <property name="title">
    <string>Details : </string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_Details">
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>normale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>équipe</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="DureDeConge">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>451</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QGroupBox {
	color: rgb(0, 0, 0);
	border:2px solid #1921ab;
	border-radius:7px;
	display:inline-block;
	cursor:pointer;
	font-family:Arial;
	font-size:11px;
	border-size:4px;
	font-weight:bold;
	font-style:italic;
	padding:0px 5px;
	text-decoration:none;
	text-shadow:2px 1px 0px #2f6627;
}</string>
   </property>
   <property name="title">
    <string>Durée de Congé : </string>
   </property>
   <widget class="QLineEdit" name="Jours">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
     border: 1px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: #C9C7F9;

 }</string>
    </property>
    <property name="placeholderText">
     <string>Saisir le nombre de jours de congé</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pour demande un demi jour taper ( 0.5 )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>plusieur jours juste taper le nombre (ex : 4)</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>440</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Temps du travail : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date début : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Num_Mat">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: #99ccff;
     selection-background-color: darkgray;
 }
.QLineEdit:hover {
	background-color: #ffffcc;
}
.QLineEdit:active {
	position:relative;
	top:1px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Taper le N° de matricule</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Valider">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
	background-color:#4579c7;
	border-radius:7px;
	display:inline-block;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:11px;
	font-weight:bold;
	font-style:italic;
	padding:0px 5px;
	text-decoration:none;
	text-shadow:2px 1px 0px #2f6627;
}
.QPushButton:hover {
	background-color:#402abd;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
